### PR TITLE
Add additional contributor name

### LINKED ISSUE #243: Add @ania-chan to "Who Built This"

### PR DESCRIPTION
**Pull request by Ania-chan**
Resolves #243

**Proposed Changes**
- Add additional contributor name to the Q&A section

**Testing Plan**
N/A

### CODE REVIEW

File: web/src/i18n/locales/en.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
         "qualifyA": "The biggest hurdle to public speaking is often our own fears. We welcome any woman with an interest in public speaking to nominate herself, regardless of training or experience.",
         "qualifyQ": "Are there prerequisites for being a speaker?",
         "title": "Frequently Asked Questions",
-        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
+        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/en.json:40]:
```suggestion
        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer, and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```

oxford comma 😇

Comment (Ania-chan) [reply web/src/i18n/locales/en.json]: @ann-kilzer added the comma- didn't know about such thing as oxford comma 😄 I'm googling it now.

**Review by ann-kilzer — APPROVED** [review]
LGTM

File: web/src/i18n/locales/en.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "qualifyA": "The biggest hurdle to public speaking is often our own fears. We welcome any woman with an interest in public speaking to nominate herself, regardless of training or experience.",
         "qualifyQ": "Are there prerequisites for being a speaker?",
         "title": "Frequently Asked Questions",
-        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
+        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer, and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/en.json:40]:
```suggestion
        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer. Current maintainers are Ann Kilzer and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```

Sorry for the back and forth... is this okay? Might be a bit more accurate 😇

Comment (ann-kilzer) [reply web/src/i18n/locales/en.json:40]: And we might want to put this in the Japanese version too

Comment (Ania-chan) [reply web/src/i18n/locales/en.json]: @ann-kilzer added!

**Pull request merged**